config/puma.rb

```diff
@@ -58,7 +58,7 @@
 
 lowlevel_error_handler do |error|
   Sentry.capture_message(error.message)
-  FileUtils.touch Rails.root.join('tmp/restart.txt')
+  # FileUtils.touch Rails.root.join('tmp/restart.txt')
 
-  [500, {}, ["Contact #{Rails.application.config.internal_mailbox} for support\n"]]
+  # [500, {}, ["Contact #{Rails.application.config.internal_mailbox} for support\n"]]
```

Comment:
@sunny-sidhu-and is it wise to keep the custom message to help identify browser errors?

Reply:
Happy to let the error bubble up as you have.